### PR TITLE
Use cwrap library from ert

### PR DESCRIPTION
**Pull request by pgdr**
This PR is the result of running sed on `s/ert.cwrap/cwrap/g`.

It reflects the upcoming changes in ert (see [ert#1159](https://github.com/Ensembles/ert/pull/1159)).

Depends on
* Ensembles/ert#1159
* Opm/opm-common#181

### CODE REVIEW

**Comment by pgdr** [comment]
This PR depends on ert#1159 but we have no integrated tests for OPM & ert, so neither travis nor jenkins will pass.

It works on my machine™ both with an external and an embedded version of the python module `cwrap`.

**Comment by joakim-hove** [comment]
jenkins build this with downstreams ert=1159 please

**Comment by pgdr** [comment]
Jenkins build failed, but after inspection it seems unrelated (Ewoms failure).

Some error "no matching function for call to"

```c++
Ewoms::ReservoirProblem<Ewoms::Properties::TTag::ReservoirBlackOilEcfvProblem>
        ::materialLawParams(int&) const
  const MaterialLawParams& matParams = problem.materialLawParams(globalDofIdx)
```

**Comment by akva2** [comment]
http://80.240.129.131:8080/job/ewoms/38/changes

**Comment by pgdr** [comment]
Well, I see now also that some tests in parser failed.  Not ready to merge yet!

**Comment by pgdr** [comment]
@joakim-hove are you sure Travis respected ert=1159?  I can't see it in the logs.  It looks like ert master was built.

**Comment by joakim-hove** [comment]
Travis does not respect ` respected ert=1159?` - but maybe you Jenkins? I will look into it.

**Comment by pgdr** [comment]
Sorry, I meant Jenkins. The following output from Jenkins
```
-- Found PythonInterp: /usr/bin/python (found version "2.7.9") 
-- Found PythonInterp: /usr/bin/python (found suitable version "2.7.9", minimum required is "2.7") 
-- Found numpy.  1.8.2 >= 1.7.1
-- Configuring done
-- Generating done
```
tells me that it's not using the latest CMake file; it should have said something about `cwrap` on line three, there.

**Comment by pgdr** [comment]
Why don't we, on a side note, build OPM with a stable ert branch/tag. We could build OPM with ert 2.0 and then this PR isn't needed.

File: opm/parser/eclipse/python/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 find_package (PythonInterp REQUIRED)
 find_package (ERTPython)
+find_package (Cwrap)
 
 if (ERT_PYTHON_PATH)
```
Comment (joakim-hove) [inline opm/parser/eclipse/python/CMakeLists.txt]: `if (CWRAP_PYTHON_PATH)` ...

**Comment by joakim-hove** [comment]
OK - this is ready to be merged when the `ERT_PYTHON_PATH` references have been removed.

**Comment by pgdr** [comment]
> OK - this is ready to be merged when the ERT_PYTHON_PATH references have been removed.

It has been removed, should we bring Jenkins along?

**Comment by joakim-hove** [comment]
jenkins build this with downstreams opm-common=181 ert=1159 please

**Pull request merged**